__tests__/app.test.js

```diff
@@ -238,4 +237,29 @@ describe('7. POST /api/reviews/:review_id/comments', () => {
   });
 });
 
+describe('8. PATCH /api/reviews/:review_id', () => {
+  test('status:202, responds with correctly updated review', () => {
+    const REVIEW_ID = 3;
+    const updateVotes = {
+      inc_votes: 5
+    };
+    return request(app)
+      .patch(`/api/reviews/${REVIEW_ID}`)
+      .send(updateVotes)
+      .expect(202)
+      .then(({ body }) => {
+        expect(body.review).toMatchObject({
+          review_id: 3,
```

Comment:
Im happy for you to use variables for REVIEW_ID, but this 3 should also but that variable as well just for consistency.